@here/harp-materials/lib/EdgeMaterial.ts

```diff
@@ -182,6 +188,9 @@ export class EdgeMaterial extends RawShaderMaterial
             depthWrite: false,
             defines
         };
+        if (params?.glslVersion === 3.0) {
+            shaderParams.glslVersion = THREE.GLSL3;
+        }
```

Comment:
If we set THREE.ShaderMaterialParameters' glslVersion directly in function "createMaterial" instead, then we don't have to repeat the same logic for every material derived from THREE.RawShaderMaterial. It's also easy to forget adding doing this whenever we add a new material.

"createMaterial" can get as parameter the renderer capabilities and set the glsl version for instances of THREE.RawShaderMaterial when the renderer is WebGL2.